src/Quasar/Mount/SparkLocal.purs

```diff
@@ -0,0 +1,58 @@
+{-
+Copyright 2016 SlamData, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module Quasar.Mount.SparkLocal
+  ( Config
+  , toJSON
+  , fromJSON
+  , parseDirPath
+  -- , toPath
+  -- , fromPath
+  , module Exports
+  ) where
+
+import Prelude
+import Data.Path.Pathy as P
+import Data.Argonaut (Json, decodeJson, jsonEmptyObject, (.?), (:=), (~>))
+import Data.Either (Either(..))
+import Data.Maybe (Maybe(..), maybe)
+import Data.Path.Pathy (Abs, Dir, Path, Sandboxed, Unsandboxed, (</>))
+import Quasar.Mount.Common (Host) as Exports
+
+type Config = Maybe (Path Abs Dir Sandboxed)
```

Comment:
What does `Nothing` means here? Seems `fromJson` simply maps `Just` over decoded value and `null` isn't handled --> probably we can omit `Maybe` wrapper